test/test_ops.py

```diff
@@ -1284,6 +1284,35 @@ def test_encode_to_tensor_long_output(self):
 
         torch.testing.assert_close(self.decode(encoded_tensor), samples)
 
+    def test_contiguity(self):
```

Comment:
Can you provide some narration in the test about what exactly we're testing? I'm afraid I'm not familiar enough with tensor manipulation to fully understand everything. For example, I'm surprised `assert_close(contiguous_samples, non_contiguous_samples)` passes when they have different strides.